src/Network/GRPC/HighLevel/Generated.hs

```diff
@@ -28,10 +28,17 @@ module Network.GRPC.HighLevel.Generated (
 
   -- * Server Auth
 , ServerSSLConfig(..)
+
+  -- * Client
```

Comment:
I usually don't bother with the version number until it is published to Hackage.  I don't make any guarantees about version numbers for work on GitHub.  For example, I might introducing a breaking change in a revision, but not bump the major version number until right before publishing to Hackage